### PR TITLE
Add the ability to pass config to nova via configmap

### PR DESCRIPTION
**Pull request by sudermanjr**
**Why This PR?**
Nova will soon support adding config via a file, which will support passing desired-versions. This will allow us to specify that when deploying the agent.

**Changes**
Changes proposed in this pull request:

* Add a configmap to nova called RELEASE-NAME-nova-config
* Mount the configmap to /config/nova.yaml
* Change the cmd for nova to be /nova find --config=/config/nova.yaml

**Checklist:**

* [x] I have updated the chart version in `Chart.yaml` following Semantic Versioning.
* [x] Any new values are backwards compatible and/or have sensible default.
* [x] Any new values have been added to the README for the Chart, or helm-docs has been run for the charts that support it.

### CODE REVIEW

File: stable/insights-agent/templates/nova/configmap.yaml [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{{- if .Values.nova.enabled -}}
+{{- $_ := set . "Label" "nova" }}
```
Comment (baderbuddy) [inline stable/insights-agent/templates/nova/configmap.yaml]: These 2 lines shouldn't be needed. If I remember right those are used by the CronJob templates.

**Review by baderbuddy — APPROVED** [review]
This feels so much cooler than what I envisioned when we were talking. I love that it's actual yaml in the values file so someone could just overwrite one of those values if they want to.

File: stable/insights-agent/templates/nova/configmap.yaml (hunk 1 above)
Comment (sudermanjr) [reply stable/insights-agent/templates/nova/configmap.yaml]: I think the top one should still be there. We don't want to create the configmap if nova isn't enabled.

But the second one I believe you are correct

**Comment by sudermanjr** [comment]
There's definitely a bug with this implementation. Seems the uploadContainer template specifies `volumes` for the whole pod, so I can't just tack it on there. 😢 

EDIT: Fixed

**Review by rbren — APPROVED** [review]

**Comment by sudermanjr** [comment]
I have tested this version in a cluster and it works exactly as intended.

**Pull request merged**